From dd94ebffbef4dc7632d8855f207d4c98b8cebf6b Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <olivier.fourdan@intel.com>
Date: Thu, 13 Mar 2014 12:11:19 +0100
Subject: Remove libxml2 global compilation flags

BZ: 177235

Remove specific per variant compilation flags for LIBXML in
various BoardConfig.mk and enable the same in xmlversion.h

Change-Id: I7bf37df8f4ff51cda043aeb2cd6b3b94d1ae455f
Category: aosp improvement
Domain: AOSP-Framework-CoreService
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: Olivier Fourdan <olivier.fourdan@intel.com>
---
 include/libxml/xmlversion.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/libxml/xmlversion.h b/include/libxml/xmlversion.h
index 10ed269..bc45af8 100644
--- a/include/libxml/xmlversion.h
+++ b/include/libxml/xmlversion.h
@@ -90,7 +90,7 @@ XMLPUBFUN void XMLCALL xmlCheckVersion(int version);
  *
  * Whether the thread support is configured in
  */
-#if 0
+#if 1
 #if defined(_REENTRANT) || defined(__MT__) || \
     (defined(_POSIX_C_SOURCE) && (_POSIX_C_SOURCE - 0 >= 199506L))
 #define LIBXML_THREAD_ENABLED
-- 
1.7.4.1

